.devcontainer/script.sh

```diff
@@ -5,9 +5,9 @@ if [ -z "$ARM_CLIENT_ID" ]; then
     exit 0
 fi
 
-# Check if $ARM_CLIENT_SECRET exists
-if [ -z "$ARM_CLIENT_SECRET" ]; then
-    echo "ARM_CLIENT_SECRET is not set"
+# Check if either $ARM_CLIENT_SECRET or $ARM_CLIENT_CERTIFICATE is set
+if [ -z "$ARM_CLIENT_SECRET" ] && [ -z "$ARM_CLIENT_CERTIFICATE" ]; then
```

Comment:
&& => and 
the comment says or => ||

Reply:
Should be correct because the bash expression is checking if both are empty. If both are empty must fail.

If both are set certificate will be preffered.

Reply:
Maybe I should write

Check if both ARM_CLIENT_SECRET or $ARM_CLIENT_CERTIFICATE are empty